pkg/watch/health.go

```diff
@@ -0,0 +1,239 @@
+package watch
+
+import (
+	"io/ioutil"
+	"net/http"
+	"time"
+
+	"github.com/square/p2/pkg/health"
+	"github.com/square/p2/pkg/kp"
+	"github.com/square/p2/pkg/logging"
+	"github.com/square/p2/pkg/pods"
+	"github.com/square/p2/pkg/preparer"
+	"github.com/square/p2/pkg/util/net"
+)
+
+// These constants should probably all be something the p2 user can set
+// in their preparer config...
+
+// number of milliseconds between reality store checks
```

Comment:
I think these comments are a bit inconsistent with the units. `3 * time.Second` appears to actually be a nanosecond count. It might just be best to say it's a Duration
http://golang.org/pkg/time/#Duration